src/main/java/uk/ac/cam/cl/dtg/segue/api/UsersFacade.java

```diff
@@ -310,6 +310,73 @@ public Response getUserPreferences(@Context final HttpServletRequest httpServlet
         }
     }
 
+    /**
+     * An endpoint for group managers (often teachers) to send a password reset request email to group members without
+     * having to know the group members account email.
+     *
+     * @param request - request information used for caching
+     * @param httpServletRequest - the request, to work ou the current user
+     * @param userIdOfInterest - userId of interest - usually a the teacher's student
+     * @return a successful response regardless of whether the email exists or an error code if there is a technical
+     *         fault
+     */
+    @POST
+    @Path("users/{user_id}/resetpassword")
```

Comment:
Can this endpoint replace the usual reset password endpoint? I suspect the only difference is the log event and who is triggering the change. It might reduce the amount of duplicated code?

Reply:
Possibly, but the reset password functionality is also available for people who are not signed in, and so would not have a user ID.

Reply:
Ah good point! Happy that it differs on that basis.